subxt/src/backend/unstable/mod.rs

```diff
@@ -294,7 +330,13 @@ impl<T: Config + Send + Sync + 'static> Backend<T> for UnstableBackend<T> {
 
         // Subscribe to the body response and get our operationId back.
         let follow_events = self.follow_handle.subscribe().events();
+
+        // `methods` already retries the call.
+        //
+        // NOTE: the sub_id could be "out-dated" if a reconnection occurs
```

Comment:
This is prob another reason to keep retry logic all in one place ie in these method impls. We can retry this whole call including getting a new subscription ID if theres any disconnection :)